src/Post/Post-List/Post-List.css

```diff
@@ -50,18 +46,16 @@
 
 .postList_note {
     font-size: 12px;
-    padding-left: 15px;
     display: inline-block;
+    padding-left: 15px;
 }
 
-.modal_post_detail{
-    position: absolute;
-    top: 30%;
-    left: 30%;
-    background:red;
-}
 
 .icon_circle {
     color: green;
     font-size: 10px;
+}
+
+.postList_name:hover, .postList_date:hover, .postList_address:hover, .postList_note:hover {
```

Comment:
Another instance where it's cleaner to use multiple class names. applying a `postList_label` to each of those divs